autogpt_platform/backend/.env.default

```
@@ -180,6 +180,7 @@ SMTP_PASSWORD=
 # Business & Marketing Tools
 APOLLO_API_KEY=
 ENRICHLAYER_API_KEY=
+AGENTMAIL_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AGENTMAIL_API_KEY key should go before the APOLLO_API_KEY key

(UnorderedKey)